src/include/network/notifiable_task.h

```diff
@@ -171,29 +160,21 @@ class NotifiableTask {
     return RegisterEvent(-1, EV_PERSIST, callback, arg);
   }
 
-    // TODO(tianyu): The original network code seems to do this as an optimization. I am leaving this out until we get numbers
-//  void UpdateEvent(struct event *event,
-//                   int fd,
-//                   short flags,
-//                   event_callback_fn callback,
-//                   void *arg,
-//                   const struct timeval *timeout = nullptr) {
-//    PL_ASSERT(!(events_.find(event) == events_.end()));
-//    if (event_del(event) == -1) {
-//      LOG_ERROR("Failed to delete event");
-//      PL_ASSERT(false);
-//    }
-//    auto result = event_assign(event, base_, fd,
-//                               flags, callback, arg);
-//    if (result != 0) {
-//      LOG_ERROR("Failed to update event");
-//      PL_ASSERT(false);
-//    }
-//    event_add(event, timeout);
-//  }
-//  void UpdateManualEvent(struct event *event, event_callback_fn callback, void *arg) {
-//    UpdateEvent(event, -1, EV_PERSIST, callback, arg);
-//  }
+// TODO(tianyu): The original network code seems to do this as an optimization. I am leaving this out until we get numbers
+  void UpdateEvent(struct event *event,
```

Comment:
Can we add documentation for all of these parameters?

Reply:
Yes